src/types/debt_kernel_error.ts

```diff
@@ -29,7 +29,19 @@ export enum DebtKernelError {
 }
 
 export namespace DebtKernelError {
-    export function parseErrors(entry: Logging.Entry): string[] {
+    export function parseLogs(logs: any[]): string[] {
+        return _.chain(logs)
```

Comment:
Fancy!  Never knew you could do this :)

Reply:
yeah, the compiler threw a tantrum initially, but hopefully having the lodash types will help going forward.